pkg/validate/dockerfile-val/testData/multiStage.json

```diff
@@ -0,0 +1,88 @@
+{
+  "dockerfile": [
+    {
+      "stage": 0,
+      "instructions": [
+        {
+          "from": [
+            "cgr.dev/chainguard/clang:latest-dev as builder"
+          ]
+        },
+        {
+          "env": [
+            "APP_HOME=/app"
+          ]
+        },
+        {
+          "run": [
+            "useradd -m -s /bin/bash -d $APP_HOME myappuser"
+          ]
+        },
+        {
+          "workdir": [
+            "$APP_HOME"
+          ]
+        },
+        {
+          "run": [
+            "apk add --no-cache clang clang-dev alpine-sdk dpkg",
+            "apk add --update --no-cache cmake",
+            "apk add --update --no-cache ccache"
+          ]
+        },
+        {
+          "copy": [
+            "src/ $APP_HOME/src/",
+            "Makefile $APP_HOME/"
+          ]
+        },
+        {
+          "run": [
+            "make -C $APP_HOME"
+          ]
+        }
+      ]
+    },
+    {
+      "stage": 1,
+      "instructions": [
+        {
+          "from": [
+            "cgr.dev/chainguard/static:latest"
+          ]
+        },
+        {
+          "env": [
+            "APP_USER=myappuser",
+            "APP_HOME=/app"
+          ]
+        },
+        {
+          "run": [
+            "useradd -m -s /bin/bash -d $APP_HOME $APP_USER"
+          ]
+        },
+        {
+          "workdir": [
+            "$APP_HOME"
+          ]
+        },
+        {
+          "copy": [
+            "--from=builder $APP_HOME/myapp $APP_HOME/myapp"
+          ]
+        },
+        {
+          "user": [
+            "$APP_USER"
+          ]
+        },
+        {
+          "entrypoint": [
+            "./myapp"
+          ]
+        }
+      ]
+    }
+  ]
+}
```

Comment:
This sign means it needs a newline in the end.

Reply:
> having a makefile target to easily run these tests on a local machine would be a great addition. 

There is a Makefile target for this specific workflow `testfunc` 
`make testfunc <function name>` 

would you like to update this target to test a specific test file instead?

And to run the test suite `make test`.  This would run `go test -v ./...`

I'll update the golintci with `revive` linter as well.
Would `golangci-lint.yaml` would be the correct place to add a test suite in GH Action or a separate Action?

Thanks

Reply:
Sounds good.